app/models/Notification.scala

```diff
@@ -17,12 +17,65 @@
 package models
 
 import org.joda.time.{DateTime, DateTimeZone}
-import play.api.libs.json.{Format, Json}
+import play.api.libs.functional.syntax._
+import play.api.libs.json._
+import reactivemongo.bson.BSONObjectID
 import uk.gov.hmrc.mongo.json.ReactiveMongoFormats
 
-case class Notification(fileReference: String, outcome: String, filename: String, createdAt: DateTime = DateTime.now.withZone(DateTimeZone.UTC))
+case class NotificationDetails(fileReference: String, outcome: String, filename: String)
+
+object NotificationDetails {
+  implicit val writes: OWrites[NotificationDetails] = Json.writes[NotificationDetails]
+  implicit val reads: Reads[NotificationDetails] =
+    ((__ \ "fileReference").read[String] and
+      (__ \ "outcome").read[String] and
+      (__ \ "filename").read[String])(NotificationDetails.apply _)
+
+  implicit val format: Format[NotificationDetails] = Format(reads, writes)
+}
+
+case class Notification(
+  _id: BSONObjectID,
```

Comment:
Nit: Not sure if it would be worth considering, but in my case I've implemented a default value of `BSONObjectID.generate()`